.github/workflows/release-tagged-dev.yml

```diff
@@ -0,0 +1,66 @@
+name: Release Tagged Dev
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'The branch, tag or SHA to checkout'
+        required: true
+        default: 'main'
+      package:
+        description: 'Comma separated list of packages to release'
+        required: true
+        type: string
+      tag:
+        description: 'The tag to use, generally a feature name'
```

Comment:
I'd argue generally a pull request number (and maybe we should even assert it but that's not a blocker)

Reply:
I guess my main issue with that is like, I don't think I can use the PR number as a ref and resolving it in that CI context seems.. hell-ish? The workflow doesn't really need to be the most human friendly given the goal is to call it via bot.

And also, there's maybe some advantages to the general `tag` when we're trying to test something special and maybe don't even have a PR.

Reply:
It doesn't have to be automatic necessarily/if its hard to do, but sure, whatever works and lets us mark a tag (that is not dev or latest lol) as deprecated